packages/react-native-win32/src/Libraries/Components/View/ViewWin32.Props.ts

```diff
@@ -187,6 +188,7 @@ export interface IViewWin32Props extends Omit<RN.ViewProps, ViewWin32OmitTypes>,
 
   accessibilitySetSize?: number;
   animationClass?: string;
+  focusable?: boolean;
```

Comment:
Do we need this if it is built-in already?

Reply:
Looks like @type/react-native still doesn't have it yet for some reason, https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native/index.d.ts, there will be some type errors if not adding it here.

Reply:
Good to know. @alloy FYI

Reply:
We should probably make a quick PR to add the typing. I can do this a bit later today while it's still on my mind.

Reply:
Sounds good, thanks!

Reply:
(Let me know if you want me to take care of it.)